docs/features/collision/collision.md

```diff
@@ -5,11 +5,137 @@ title: Collision
 description: 2D grid-based collision system.
 ---
 
-:::warning[Not up to date]
-This page **is not** up to date for MonoGame.Extended `@mgeversion@`.  If you would like to contribute to updating this document, please [create a new PR](https://github.com/craftworkgames/craftworkgames.github.io/pulls)
+:::tip[Up to date]
+This page is **up to date** for MonoGame.Extended `@mgeversion@`.  If you find outdated information, [please open an issue](https://github.com/craftworkgames/craftworkgames.github.io/issues).
 :::
 
-## Usage
+## Collision basics
+
+To use the Collision code, you need to perform the following 4 steps:
+1. Create a class that implements the `ICollisionActor` interface.
+1. Implement the `OnCollision` method in your class.  This defines what happens when something hits your object.
+1. Create a `CollisionComponent` instance defining the bounds of your world.
+1. `Insert` an instance of your class into the CollisionComponent.
+
+The [Full Example](#full-example) uses an extra Interface IEntity that's optional but can be helpful.
+
+## Detailed Explanation
+
+There are a few different items that belong to the collision Namespace.
+1. Space Algorithms
+1. Layer
+1. CollisionComponent
+1. ICollisionActor
+
+You don't need to understand them in detail to use them.  Below are high level explanations of each.
+
+You may skip to the [Full Example](#full-example) section and ignore these details if you wish.
+
+### Space Algorithms
+
+(Advanced/Optional: You don't need to create one of these, this is just explaining how things work behind the scenes.  Skip to the [Full Example](#full-example) for the bare minimum.)
```

Comment:
Given the note concerning moving the Layer section down, this clarification wouldn't be needed anymore.  Same for the other sections that begin with clarifications on if something is required or optional.  Start with all required things before the example, then do the optional things after.